modules/client/examples/05-encoders-decoders/13-addresslist-voting/03-remove-members.ts

```diff
@@ -17,37 +17,42 @@ import { addresslistVotingClient } from "./index";
 const members: string[] = [
   "0x1357924680135792468013579246801357924680",
   "0x2468013579246801357924680135792468013579",
-  "0x0987654321098765432109876543210987654321"
+  "0x0987654321098765432109876543210987654321",
 ];
 
 const pluginAddress: string = "0x0987654321098765432109876543210987654321"; // the address of the AddresslistVoting plugin contract installed in the DAO
 
-const action: DaoAction = addresslistVotingClient.encoding.removeMembersAction(pluginAddress, members);
+const action: DaoAction = addresslistVotingClient.encoding.removeMembersAction(
+  pluginAddress,
+  members,
+);
 console.log({ action });
 
 /* MARKDOWN
 Returns:
 
 ```json
-{
-  to: "0x1234567890...",
-  value: 0n,
-  data: Uint8Array[12,34,45...]
+{ action:
+  {
+    to: "0x1234567890...",
+    value: 0n,
+    data: Uint8Array[12,34,45...]
+  }
 }
 ```
 
 ### Decoding
 */
 
-
-const removedMembers: string[] = addresslistVotingClient.decoding.removeMembersAction(action.data);
-console.log({ removedMembers });
+const decodedParams: string[] = addresslistVotingClient.decoding
+  .removeMembersAction(action.data);
+console.log({ decodedParams });
 
 /* MARKDOWN
 Returns:
 
 ```
-{ removedMembers:
+{ decodedParams:
```

Comment:
Why dropping a meaningful name?

Reply:
Because the variable was named `decodedParams`, it didn't make sense to use `removedMembers`. Additionally, in all the other examples, the decoded action variable is also called `decodedParams`, so having one variable with a specific name and all the others without it didn't make much sense to me.